app/main.py

```diff
@@ -297,14 +308,16 @@ def grab_secrets(self):
             if not os.path.exists(folder):
                 os.makedirs(folder)
 
-        client = self._get_client()
+        credential = self._get_credential()
+        secret_client = SecretClient(vault_url=vault_base_url, credential=credential)
+        certificate_client = CertificateClient(vault_url=vault_base_url, credential=credential)
         _logger.info('Using vault: %s', vault_base_url)
 
         # Retrieving all secrets from Key Vault if specified by user
         if secrets_keys is None:
             _logger.info('Retrieving all secrets from Key Vault.')
 
-            all_secrets = list(client.get_secrets(vault_base_url))
+            all_secrets = list(secret_client.list_properties_of_secrets())
```

Comment:
Just making sure, since we're using some different methods to get / list secrets - these updates won't require any changes to the access policies needed by the keyvault-agent, right?

Reply:
No changes to access policy.